.github/workflows/build_symengine.yml

```diff
@@ -167,3 +167,29 @@ jobs:
     - name: upload package
       if: github.event_name == 'push'
       run: conan upload symengine/${{ env.SYMENGINE_VER }}@tket/stable --all -r=tket-conan
+  manylinux:
+    name: build symengine (manylinux)
+    needs: changes
+    if: needs.changes.outputs.recipes_symengine == 'true'
+    runs-on: ubuntu-20.04
+    env:
+      UPLOAD_PACKAGE: "NO"
```

Comment:
I am not fully convinced that this is "working as expected", but if this is not working it will not effect this workflow, an empty variable is still not yes.

(I think this is only working within this file, not in the bash script. )

Reply:
It is passed to the docker image via the file `env-vars` below.